### PR TITLE
Fix issue 1335

### PR DESCRIPTION
**Pull request by ruanitto**
## Pull Request (PR) Checklist
Please check if your pull request fulfills the following requirements:
- [X] The PR is submitted to the `develop` branch.
- [X] I understand the **[Ivy Developer Guidelines](../docs/Developer-Guidelines.md)**.
- [X] I've read the **[Contribution Guidelines](https://github.com/ILIYANGERMANOV/ivy-wallet/blob/main/CONTRIBUTING.md)**.
- [X] The code builds and is tested on an actual Android device.
- [X] I confirm that I've run the code locally and everything works as expected.
- [X] I confirm that I've run Ivy Wallet's UI tests (`androidTest`) and all tests are passing
  successfully.

_Important: Don't worry if you experience flaky UI tests. Just re-run the failed ones again and if they pass => it's all good!_

_Put an `x` in the boxes that apply._
- [x] Demo: Checking checkbox using `[x]`

## Pull Request Type
Please check the type of change your PR introduces:

- [ ] Bugfix
- [X] Feature
- [ ] Code style update (formatting, new lines, etc.)
- [ ] Refactoring (no functional changes, renaming)
- [X] Small improvement (fix typo, UI fine-tune, change color or something small)
- [ ] Gradle Build related changes
- [ ] Dependencies update (updating libraries)
- [ ] Documentation (clarifying comments, KDoc)
- [ ] Tests (Unit, Integration, UI tests)
- [X] Other (please describe): Translation to ptBR

_Put an `x` in the boxes that apply._

## Does this PR closes any GitHub Issues?
Check **[Ivy Wallet Issues](https://github.com/ILIYANGERMANOV/ivy-wallet/issues)**.
- Closes #N/A (type issue number here)

## What's changed?
Describe with a few bullets **what's new:**
- Included translate to ptBR 

## How to run Ivy Wallet's UI tests (`androidTest`)
**Connect Android Emulator**
- Pixel 5 API 29+ AVD emulator _(recommended)_
- Pixel 3XL API 29+ AVD emulator _(recommended)_
- Large screen physical device _(might also work)_

**Method 1: Android Studio UI**
- Find `com (androidTest)` package
- Right click
- `Run 'Tests in 'com''`

_Note: If you've checked "Compact Middle Packages" the option will appear as `com.ivy.wallet (androidTest)`._

**Method 2: Gradle Wrapper**
- `chmod +x gradlew` (Linux)
- `./gradlew connectedDebugAndroidTest`

**Method 3: Fastlane**
- Install Ruby 2.7
- `bundle install`
- `bundle exec fastlane ui_tests`

### CODE REVIEW

**Comment by ILIYANGERMANOV** [comment]
That's awesome!! Thank you! Once the CI checks pass, I'll merge it 👍

**Comment by ILIYANGERMANOV** [comment]
@ruanitto remove "produto" string and it should be okay

First failure:

/home/runner/work/ivy-wallet/ivy-wallet/resources/src/main/res/values-pt/strings.xml:248: Error: "produto" is translated here but not found in default locale [ExtraTranslation]
    <string name="produto">Produto</string>

File: resources/src/main/res/values-pt/strings.xml [hunk 1]
```diff
@@ -0,0 +1,440 @@
...
+    <string name="import_data">Importar dados</string>
+    <string name="app_settings">Configurações do aplicativo</string>
+    <string name="lock_app">Bloquear o aplicativo</string>
+    <string name="show_notifications">Mostrar notificações</string>
+    <string name="hide_balance">Ocultar saldo</string>
+    <string name="hide_balance_description">Clique no saldo oculto para mostrar o saldo por 5 segundos</string>
+    <string name="other">Outro</string>
+    <string name="rate_us_on_google_play">Avalie-nos no Google Play</string>
+    <string name="share_ivy_wallet">Compartilhar carteira Ivy</string>
+    <string name="produto">Produto</string>
```
Comment (ILIYANGERMANOV) [inline resources/src/main/res/values-pt/strings.xml:248]: Remove this because it fails lint.

Comment (ruanitto) [reply resources/src/main/res/values-pt/strings.xml]: Thanks! It's now fix.

**Comment by code-a1** [comment]
Fix #1335

**Pull request merged**